erpnext/projects/doctype/project_template_task/project_template_task.json

```diff
@@ -49,10 +52,23 @@
    "fieldtype": "Link",
    "label": "Assigned User",
    "options": "User"
+  },
+  {
+   "fieldname": "column_break_4",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "parent_task",
+   "fieldtype": "Int",
+   "label": "Parent Task"
```

Comment:
- Lets make this field a `select` field and the values will be `row.idx + row.subject` this will make it easier for the user to select it rather than using the `int` fieldtype.
- While creating tasks in the backend, you can get the `idx` from the value of `row.parent`

Reply:
I like this idea